howto/enable-dapr-api-token-based-authentication/README.md

```diff
@@ -0,0 +1,93 @@
+# Enable Dapr APIs token-based authentication
+
+By default, Dapr relies on the network boundary to limit access to its public API. If you plan on exposing the Dapr API outside of that boundary, or if your deployment demands an additional level of security, consider enabling the token authentication for Dapr APIs. This will cause Dapr to require every incoming gRPC and HTTP request for its APIs for to include authentication token, before allowing that request to pass through. 
+
+## Creation of token
```

Comment:
Suggestion to change to "Create a token"

Reply:
done